index.css

```diff
@@ -0,0 +1,159 @@
+body {
+    margin: 0;
+    padding: 0;
+    background-color: #f6f9fc;
+    color: #444;
+    font-size: 14px;
+    font-weight: 400;
+    font-family: "Segoe UI" "Helvetica Neue", sans-serif;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+    line-height: 1.72;
+  }
+  .navbar {
+    background-color: #32325d;
+    height: 5rem;
+    display: flex;
+    justify-content: center; /* Center vertically */
+    align-items: center; /* Center horizontally */
+  }
+  .logo {
+    color: white;
+    position: absolute;
+    left: 2rem;
+  }
+  
+  .navbar_content {
+    line-height: 17px;
+    display: flex;
+    gap: 50px;
+  }
+  .fa-solid {
+    color: white;
+  }
+  .container .title {
+      font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
+    background-color: white;
+    height: 5rem;
+    font-size: 33px;
+    font-weight: bold;
+    letter-spacing: -1px;
+    color: #222;
+    line-height: 1.2;
+    padding-top: 20px;
+    padding-left: 15px;
+  }
+  .navbar a {
+    color: #8789cd;
+    font-size: 13px;
+    font-weight: 600;
+    line-height: 21px;
+    text-transform: uppercase;
+    text-decoration: none;
+  }
+  .container {
+    width: 100%; /* Set the width to 100% of its containing element */
+    max-width: 1200px; /* Limit the maximum width of the container */
+    margin: 0 auto; /* Center the container horizontally */
+    padding: 20px; /* Add padding inside the container */
+  }
+  .content {
+    margin-top: 40px;
+    display: flex;
+    gap: 10px;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
+  .box {
+    width: 31%;
+  }
+  
+  .content .list {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
+  
+  .content img {
+    width: 100%;
+    height: 220px;
+      
+  }
+  @media (max-width: 768px) {
```

Comment:
Put the media query at the bottom of the page or nested inside the class that you want to make responsive.